client/src/css/Table.css

```diff
@@ -0,0 +1,78 @@
+
+.table-logo {
+  width: 100px;
+  height: 100px;
+}
+.table-text-logo {
+  width: 350px;
+  height: 10vh;
+}
+.hr{width: 100px;
+    border: 1px black solid;
+
+}
+@media only screen and (max-width: 414px){
+    .hr{
+        width: 70px;
+    }
+}
+<<<<<<< HEAD
```

Comment:
It looks like there's a merge conflict tag in here - we just need to rebase off of master, I can add some instructions in my overall comment on the typical strategy to do that!

Reply:
I did push, not sure if that what you wanted, let me know